src/Generators/Webserver/Database/DatabaseGenerator.php

```diff
@@ -35,14 +35,21 @@ class DatabaseGenerator
      */
     protected $mode;
 
+    /**
+     * @var DatabaseDriverFactory
+     */
+    protected $databaseDriverFactory;
+
     /**
      * DatabaseGenerator constructor.
      * @param Connection $connection
+     * @param DatabaseDriverFactory $databaseDriverFactory
      */
-    public function __construct(Connection $connection)
+    public function __construct(Connection $connection, DatabaseDriverFactory $databaseDriverFactory)
```

Comment:
We're in the database generator class, for clarity I'd use `$factory` here. No need to overcomplicate the variable in this scope. Sorry for being a nitpick 😊

Reply:
No idea why it marked outdated already, but please oblige me @Sti3bas 👍

Reply:
Done.